### PR TITLE
pep-0557.rst: typos, .. code:: python directives

### PR DESCRIPTION
**Pull request by westurner**
@ericvsmith

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: pep-0557.rst [hunk 1]
```diff
@@ -186,10 +194,10 @@ signature.  That is, these three uses of ``@dataclass`` are equivalent::
 
 The parameters to ``dataclass`` are:
 
-- ``init``: If true (the default), a ``__init__`` method will be
```
Comment (ericvsmith) [inline pep-0557.rst]: I always read these as "a dunder init method", so the original sounds correct to me.

**Comment by westurner** [comment]
I've already signed the CLD. And now I've added my github username to the bpo account.

File: pep-0557.rst (hunk 1 above)
Comment (westurner) [reply pep-0557.rst]: Fair enough.

**Comment by ericvsmith** [comment]
I don't see any uses of "code:: python" in any existing PEP, so I'm not inclined to start using it here.

**Comment by ericvsmith** [comment]
Meanwhile, I've fixed the two typos with extra s's that you noted. Thanks.

**Comment by brettcannon** [comment]
No other PEPs have language directives because we have not had Pygments installed on www.python.org. I know someone was trying to make that happen, but I don't know where that ended up.

**Comment by Mariatta** [comment]
In https://github.com/python/pythondotorg/pull/1063#issuecomment-305972905 it was said that we'll be moving to Sphinx/Readthedocs so we won't be adding Pygments.

**Comment by gvanrossum** [comment]
I'm guessing this isn't going to be merged, so closing.

**Comment by westurner** [comment]
Just baffling how this occurs.

If I had to guess? Prolly 2-3 years before we get syntax highlighting on
these bad boys

**Comment by gvanrossum** [comment]
That was a real helpful comment, Wes.

**Pull request closed without merging**